content/docs/command-reference/status.md

```diff
@@ -33,18 +34,20 @@ options:
 | remote | `--remote`     | Comparisons are made between the cache, and the given remote. Remote storage is defined using the `dvc remote` command.     |
 | remote | `--cloud`      | Comparisons are made between the cache, and the default remote, typically defined with `dvc remote --default`.              |
 
-DVC determines which data and code files to compare by analyzing all
-[DVC-files](/doc/user-guide/dvc-file-format) in the <abbr>workspace</abbr> (the
-`--all-branches` and `--all-tags` options compare multiple workspace versions).
+DVC determines which data and code files to compare by analyzing all stages (in
+[`dvc.yaml`](/doc/user-guide/dvc-file-format)) and
+[`.dvc` files](/doc/user-guide/dvc-file-format) in the <abbr>workspace</abbr>
+(the `--all-branches` and `--all-tags` options compare multiple workspace
+versions).
 
-The comparison can be limited to certain DVC-files only, by listing them as
-`targets`. (Changes are reported only against these.) When this is combined with
-the `--with-deps` option, a search is made for changes in other stages that
-affect each target.
+The comparison can be limited to certain stages (in `dvc.yaml`) and `.dvc` files
+only, by listing them as `targets`. (Changes are reported only against these.)
+When this is combined with the `--with-deps` option, a search is made for
+changes in other stages that affect each target.
 
-In the `local` mode, changes are detected through the hash value of every file
-listed in every DVC-file in question against the corresponding file in the file
-system. The command output indicates the detected changes, if any. If no
+In the local mode, changes are detected through the hash value of every file
+listed in every `.dvc` file in question against the corresponding file in the
```

Comment:
Lets do this here:
"listed in every stage (in `dvc.yaml` or `.dvc` files) in..."